presto-tpch/src/main/java/com/facebook/presto/tpch/statistics/ColumnStatisticsRecorder.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tpch.statistics;
+
+import java.util.Optional;
+import java.util.TreeSet;
+
+class ColumnStatisticsRecorder
```

Comment:
This class goes in pair with TableStatisticsRecorder, which is not just a simple builder. Also, the method names are made to comply with the 'recorder'. Also, it's disputable if the (tiny, but still) logic in this class' methods qualifies it to be just 'a builder'.